.pre-commit-config.yaml

```diff
@@ -198,15 +198,15 @@ repos:
 
       - id: schemas
         name: Build AVD schemas and documentation.
-        entry: sh -c 'exec python-avd/scripts/build-schemas.py'
+        entry: sh -c 'exec python-avd/scripts/build_schemas.py debug'
```

Comment:
Is is on purpose you still have debug here?

Reply:
yes to see the same things as before